packages/npm-packages/ruby-wasm-wasi/src/browser.script.ts

```diff
@@ -16,17 +16,35 @@ export const main = async (pkg: { name: string; version: string }) => {
 };
 
 const runRubyScriptsInHtml = async (vm) => {
```

Comment:
It's not directly related to your change, but we need to consider when the `text/ruby` scripts should be scanned.
[`DOMContentLoaded`](https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event) looks more suitable to me than the current immediate scan.

Could you update it to defer script evals after the event if you don't mind?

Reply:
Delayed execution of runRubyScriptsInHtml at https://github.com/ruby/ruby.wasm/pull/53/commits/3c4ef9e253914225e21d9c49f07287001e084eae .
Also, added support for cases where the DOMContentLoaded event has already been fired.

Reply:
Thanks so much!